src/com/sun/ts/tests/jaxws/wsa/w2j/document/literal/typesubstitution/custom-client.xml

```diff
@@ -18,7 +18,7 @@
 -->
 
 <bindings wsdlLocation="WSAW2JDLTypeSubstitutionTest.wsdl"
-         xmlns="http://java.sun.com/xml/ns/jaxws" 
+         xmlns="https://jakarta.ee/xml/ns/jaxws"
          xmlns:wsdl="http://schemas.xmlsoap.org/wsdl/" 
          xmlns:xsd="http://www.w3.org/2001/XMLSchema" 
          xmlns:jaxb="http://java.sun.com/xml/ns/jaxb">
```

Comment:
`xmlns:jaxb="http://java.sun.com/xml/ns/jaxb` needs to be changed to `xmlns:jaxb="https://jakarta.ee/xml/ns/jaxb` too